Modules/xASL_module_ASL.m

```diff
@@ -415,11 +420,22 @@
 		% Note: The paths x.P.(Pop_)Path_PWI4D_used can differ from a PWI4D obtained from a simple subtraction as we may have merged sessions,
 		% and the user or ExploreASL may have removed volumes from PWI4D. That's why we save and use x.P.(Pop_)Path_PWI4D_used in memory, such
 		% that the correct NIfTI will be used later in the pipeline (e.g., in xASL_wrp_VisualQC)
+
+		% Check again if we have multiple values for certain ASL parameters (PLD, LD, TE)
+		fprintf('%s\n','Sessions were concatenated. Checking again if LD, PLD, TE have multiple values...');
```

Comment:
Yes. I didn't do this previously because we indeed use x.P.Path_PWI4D_used or x.P.Path_ASL4D. And we passed jsonPath to the function. But I can simplify and unify this. Indeed a good idea to put all things inside. Fixed in the last commit.